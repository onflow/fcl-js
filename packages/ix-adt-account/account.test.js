const {account} = require("./account.cjs.js")

test("Snapshot of empty account structure", () => {
  expect(account()).toMatchSnapshot()
})
