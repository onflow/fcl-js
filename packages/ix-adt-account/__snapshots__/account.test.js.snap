// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot of empty account structure 1`] = `
Object {
  "addr": null,
  "keyId": null,
  "kind": "ACCOUNT",
  "resolve": null,
  "roles": Object {
    "authorizer": false,
    "param": false,
    "payer": false,
    "proposer": false,
  },
  "sequenceNum": null,
  "signature": null,
  "signingFunction": null,
  "tempId": null,
}
`;
